openedx/features/caliper_tracking/tests/expected/openassessmentblock.get_peer_submission.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Started",
```

Comment:
Ideal action in this case would be `Reviewed`, but since that is deprecated. I believe `Viewed` is the proper action.

Reply:
Done. I was thinking about it but forgot to update it (Y)